codegen-sbt/src/sbt-test/codegen/test-compile/project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 sys.props.get("plugin.version") match {
   case Some(x) => addSbtPlugin("com.github.ghostdogpr" % "caliban-codegen-sbt" % x)
   case _       => sys.error("""|The system property 'plugin.version' is not defined.
-                               |Specify this property using the scriptedLaunchOpts -D.""".stripMargin)
+                         |Specify this property using the scriptedLaunchOpts -D.""".stripMargin)
```

Comment:
This change is also quite bad for readability :(

Reply:
In my experience, fixing the "ugly" cases like this is best done manually. For instance 

```scala
  case _ => 
    val msg = """..."""
    sys.error(msg)
```